_unittest/test_Circuit.py

```diff
@@ -175,3 +179,23 @@ def test_19_create_EyE_setups(self):
         setup_name = "Dom_AMI"
         assert self.aedtapp.create_setup(setup_name,"NexximAMI")
         pass
+
+    def test_20_create_AMI_plots(self):
+        self.aedtapp.load_project(self.ami_example_project, close_active_proj=True)
+        assert self.aedtapp.post.create_ami_initial_response_plot("AMIAnalysis", "b_input_15", self.aedtapp.available_variations.nominal,
+                                                      plot_type="Rectangular Stacked Plot", plot_final_response=True,
+                                                      plot_intermediate_response=True, plotname="MyReport") == "MyReport"
```

Comment:
```suggestion
        report_name = "MyReport"
        assert self.aedtapp.post.create_ami_initial_response_plot("AMIAnalysis", "b_input_15", self.aedtapp.available_variations.nominal,
                                                      plot_type="Rectangular Stacked Plot", plot_final_response=True,
                                                      plot_intermediate_response=True, plotname=report_name) == report_name
```
Best to make it clear which variables we expect to be identical.